### ISSUE TITLE
Opening lammps atom files fails

### ISSUE THREAD

**Issue by merrygoat** [tranqui/MDMAmazing#3]
Opening an atom file with the snapshot reader fails in atom.py. 

Atom files store particle coordinates as a fraction of the box size, to convert the coordinates from fractional to real the code multiplies the coordinates by the box size. 

The problem is that the code reads the box boundaires as -x, +x, etc. and then tries to multiply the coordinates by this tuple. The code should add the bounds to get the total box size and then multiply by this.

Perhaps something like this?
`

                # Item containing the bounding box.
                elif ' '.join(item[1:3]) == 'BOX BOUNDS':
                    d = len(item[3:])
                    self.x = numpy.empty((self.n, d))
                    self.box = numpy.zeros(d, dtype=numpy.longdouble)

                    for c in range(d):
                        boundary = f.readline().split()
                        for bound in boundary:
                            self.box[c] += abs(float(bound))
`

**Comment by tranqui** [comment]
Good spot @merrygoat . This is a remnant from when we used box to indicate the lengths of the box: now we differentiate between the complete box description (i.e. left and right in each dimension) and the dimensions of the box (i.e. right - left). The former is labelled box, whilst the latter is labelled box_dimensions.

I have updated the [examples](https://tranqui.github.io/MDMAmazing/build/html/examples.html#creating-and-a-lammps-simulation) to show how to read and write snapshots in LAMMPS. Note the tests at the end of the examples: these all pass on my system.

Fixed in dc41d7c663df81e1b956898e99bfad6e5f4b0b2c.
